<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sampleDialog</class>
 <widget class="QDialog" name="sampleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>233</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>243</width>
    <height>233</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>243</width>
    <height>233</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Sample</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>res/logo.png</normaloff>res/logo.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameTxt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>NDTXX</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>14</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Top Left</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tlTxt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>[0,4]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Top right</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="trTxt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>[3.7,4]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bottom left</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="blTxt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>[0,0]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bottom right</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="brTxt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>[4,0]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cTxt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>[4,3.5]</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>sampleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>sampleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
